### ISSUE TITLE
Simplified Chinese font is missing in i18n NUX screenshots on CI.

### ISSUE THREAD

**Issue by southp** [Automattic/wp-e2e-tests#1075]
Please see the screenshot:
![image](https://user-images.githubusercontent.com/1842898/37807724-e574b786-2e81-11e8-9f6c-74bd44963dd7.png)

It seems to be something similar to https://github.com/Automattic/wp-e2e-tests/issues/645. @hoverduck Do you think this is something fixable by the same approach as https://github.com/Automattic/wp-e2e-tests-i18n/pull/4?

**Comment by hoverduck** [comment]
Yeah, that's likely going to require a similar fix.  @rachelmcr I lost track, where do the i18n NUX tests run, is that in a wrapper repo or here in the main one?  Wherever they run we'll need to add a step to the CircleCI config file to install the fonts.

**Comment by rachelmcr** [comment]
The i18n NUX tests run in `wp-e2e-tests-i18n` so we should be able to install the necessary Chinese fonts as we did with the Japanese fonts here:

https://github.com/Automattic/wp-e2e-tests-i18n/blob/master/circle.yml#L17

@southp Do you know which font(s) are needed? I can look into that but I figured I'd check in case you can make a recommendation for that.

**Comment by rachelmcr** [comment]
I tried a fix that installs `fonts-noto-cjk` but I get an error:

> E: Unable to locate package fonts-noto-cjk

I'm not too familiar with installing fonts like this so I'm not sure if I'm missing a step (I've already tried adding `sudo apt-get update` first, but no luck) or if I need to use a different font.

**Comment by hoverduck** [comment]
I have no idea if it's the right font package or not, but I was able to `sudo apt-get install fonts-wqy-zenhei` (based off of [this link](https://askubuntu.com/questions/490829/how-can-i-install-chinese-fonts-on-kubuntu-14-04) that says it's a "complete Chinese font")

**Comment by southp** [comment]
@hoverduck I don't have much experience with kubuntu, but`fonts-wqy-zenhei` looks quite promising. Is there a way that I can generate the same screenshots locally on my own? I'd love to help verify :)

**Comment by alisterscott** [comment]
I believe this should do the trick: https://github.com/Automattic/wp-e2e-tests-i18n/pull/6

**Comment by alisterscott** [comment]
@rachelmcr where do these screenshots end up? I forget and I can't remember where to look. Do you know if there's doco on these i18n tests and screenshots?

**Comment by alisterscott** [comment]
I think that PR is picking up the font now, although some of the i18n scenarios are failing.

https://463-86588833-gh.circle-artifacts.com/0/home/ubuntu/wp-e2e-tests-i18n/wp-e2e-tests/screenshots/ZH-CN-DESKTOP--1522222125539-should-redirect-to-the-correct-url-for-wordpress-com--zh-cn-.png

![zh-cn-desktop--1522222125539-should-redirect-to-the-correct-url-for-wordpress-com--zh-cn-](https://user-images.githubusercontent.com/128826/38015192-5d8686c0-32ae-11e8-941f-4ca231ea02d2.png)

**Comment by rachelmcr** [comment]
>where do these screenshots end up? I forget and I can't remember where to look. Do you know if there's doco on these i18n tests and screenshots?

Failures are sent to #i18n-bots in Slack and reviewed by Team Global directly.

I don't believe these tests are documented. They were added in https://github.com/Automattic/wp-e2e-tests/pull/635 but the `-I` flag was never documented — I'll at least add that flag to our README for reference. (Also worth noting that these are different from the tests I set up with the `-i` flag, which I don't believe are being used at the moment.)
